app/views/tran_details/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<p style="color: green"></p>
+<h2>GROUP TRANSACTIONS</h2>
+<% @group_name = TranGroup.find(params[:tran_group_id]) %>
+<h5><strong>Group Name: </strong> <%= @group_name.groupName %></h5>
+<h5 style="color:skyblue"><strong>Group Total :</strong> <%= @total_value %></h5>
+<div id="tran_details">
+  <table class="table table-hover">
+    <thead>
+      <tr>
+        <th scope="col">Date</th>
+        <th scope="col">Transection Details</th>
+        <th scope="col">Amount</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @tran_details.each do |tran|%>
+        <tr>
+          <td scope="row"><%= tran.created_at.strftime('%d-%m-%y')%></td>
+          <td scope="row"><%= tran.itemDetails %></td>
+          <td scope="row"><%= tran.amount %></td>
+        </tr>
+      <%end%>
+    </tbody>
+  </table>
+</div>
+<%= link_to "Create Transaction", new_tran_group_tran_detail_path, class: "btn btn-outline-success" %>
+<%= link_to "Back Index", tran_groups_path, class: "btn btn-outline-warning" %>
```

Comment:
- [ ] Please once more kindly adhere to the provided design guidelines. This button should be at the bottom :+1:

Reply:
Resolved